test/integration/tests/fixes/38072036.mjs

```diff
@@ -0,0 +1,34 @@
+const testSteps = [
+  (chart) => {
+    const data = {
+      series: [
+        { name: 'Foo', values: ['Alice', 'Bob', 'Ted'] },
+        { name: 'Bar', values: [15, 32, 12] }
+      ]
+    }
+    return chart.animate({ data }, 0)
+  },
+  (chart) =>
+    chart.animate(
+      {
+        config: {
+          x: 'Foo',
+          y: ['Foo', 'Bar']
+        }
+      },
+      1
+    ),
+  (chart) => chart.animate({}, 2),
```

Comment:
Unfortunately the runtime of the animation won't be checked here, since the test runner just pauses the anim and seeks into it in different percents.

Reply:
@veghdev if we would use duration in seek() instead of percent, then we could detect animation speed related changes.

Reply:
created: https://github.com/orgs/vizzuhq/projects/10/views/1?pane=issue&itemId=40614195